src/index.js

```diff
@@ -27,25 +27,29 @@ export function rangeFromLineNumber(textEditor: TextEditor, line: number, column
     throw new Error(`Line number (${lineNumber}) greater than maximum line (${lineMax})`)
   }
 
-  let colStart = column
-  if (!Number.isFinite(colStart) || Number.isNaN(colStart) || colStart < 0) {
-    const indentation = buffer.lineForRow(lineNumber).match(/^\s+/)
-    if (indentation && indentation.length) {
-      colStart = indentation[0].length
-    } else {
-      colStart = 0
-    }
+  const columnGiven = typeof column === 'number' && Number.isFinite(column) && column > -1
```

Comment:
Why did you change the "check if it's a number" logic? If this is somehow better, then the logic on line 19 should be updated as well.